_scripts/api-reference.sh

```diff
@@ -2,21 +2,33 @@
 
 set -e
 
-API_VERSION="$(cat .api-version)"
+API_VERSION="develop"
 TARBALL_URL="https://codeload.github.com/tinymce/tinymce/tar.gz/$API_VERSION"
```

Comment:
Interesting. This suggests we don't need published releases to compile API docs against, as long as the API version tag contains code with the correct docs.

Reply:
There's also a build from local source flavour _scripts/api-reference-local.sh thats wired into yarn build-local ../path/to/local/tinymce

Reply:
yeah I saw that, I meant in reference to a conversation yesterday about when source code had to be changed for API docs.